plugin.xml

```diff
@@ -50,7 +50,11 @@
     <config-file target="AndroidManifest.xml" parent="/manifest/application">
       <service android:name="org.jboss.aerogear.android.authorization.oauth2.OAuth2AuthzService"/>
     </config-file>
+    <config-file target="AndroidManifest.xml" parent="/manifest/application">
+      <meta-data android:name="com.google.android.gms.version" android:value="@integer/google_play_services_version" />
```

Comment:
Does anyone know the significance of this entry?  What effect will it have if google play services is not available?  Hopefully it will be safely ignored.